src/containers/ConfirmPassword/confirmPass.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Input, Icon, Button, Form } from 'antd';
+import { ReactComponent as ConfirmImg } from '../assets/images/confirmPass.svg';
+import Header from '../../components/Header';
+
+import './confirmPass.css';
+
+const confirmPass = props => {
+  const {
+    history: { goBack },
+    form: { getFieldDecorator, validateFieldsAndScroll },
+  } = props;
+
+  const handleSubmit = e => {
+    e.preventDefault();
+    validateFieldsAndScroll((err, values) => {
+      if (!err) {
+        // eslint-disable-next-line no-console
+        console.log('Received values of form: ', values);
+      }
+    });
+  };
+
+  return (
+    <div className="confirm-pass">
+      <Header text="Confirm Password" handleBack={goBack} />
+
+      <section className="confirm-pass__body">
+        <p className="confirm-Password__text">
```

Comment:
```suggestion
        <p className="confirm-pass__text">
```

Reply:
Will need to make the same change at confirmPass.css #9, #38, #71, #105 as well